apps/sap-commerce-cloud/frontend/src/AppConfig/AppConfig.tsx

```diff
@@ -82,9 +83,13 @@ export default function AppConfig({
 
   useEffect(() => {
     (async () => {
+      const cma = createClient({ apiAdapter: sdk.cmaAdapter });
```

Comment:
followed this way of doing things from another app but if there's a better way to rewrite this let me know!

Reply:
nope, this is a great way to do it